app/models/plate_creation.rb

```diff
@@ -7,6 +7,7 @@ class PlateCreation < AssetCreation
   # This is the child that is created from the parent.  It cannot be assigned before validation.
   belongs_to :parent, class_name: 'Plate'
   attr_accessor :barcode
```

Comment:
Can barcode be removed?

Reply:
I didn't think it could as it seemed like it was tied into the same named attribute on another class, but I've just done so and all tests still pass.  I did have to update a factory.